<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_Main</class>
 <widget class="QMainWindow" name="C_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MovingLab</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_ShowCoreSlam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show CoreSlam HMI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ShowCommand">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Commands</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ShowScanseLidar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Scanse Lidar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ShowPathFinder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Path Finder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ShowSlamwareLidar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Slamware Lidar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ShowCoreSlamPF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show CoreSlamPF HMI</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
